.github/workflows/backend-ci.yml

```diff
@@ -24,18 +24,15 @@ concurrency:
 jobs:
   build-test:
     runs-on: ubuntu-latest
-    if: github.event_name != 'pull_request' || github.event.pull_request.head.repo.full_name != github.repository # Skip the launch if it has already been launched by the push trigger
-    strategy:
-      matrix:
-        python-version: ["3.13"]
+    #if: github.event_name != 'pull_request' || github.event.pull_request.head.repo.full_name != github.repository # Skip the launch if it has already been launched by the push trigger
```

Comment:
@inimaz J'oublie pourquoi on l'a fait exactement. Cela dit "skip launch if it ahas already been launched" mais il m'a l'air un peu bizarre...

J'ai l'impression que quand je change juste le backend-ci.yml que le backend-ci ne tourne pas...

Il est peut-etre le "on" qui fait tourner 2 fois...pourquoi pas juste faire `on: [pull_request]` ?

Reply:
Cette condition avait-elle pour but d'éviter de lancer le workflow lors d'un push sur un fork de ce repository ?